zh-CN/mapping_dev.zh_CN

```diff
@@ -25,7 +25,7 @@
 
 *.btn"Cancel"=取消
 *.btn"Close"=关闭
-*.btn"OK"=好
+*.btn"OK"=同意
```

Comment:
Regarding this,  I think need to cancel the constant setting. 
Because it has different meanings in different scenarios,
So at the same time I am also discussing with   @mstrap  .

I see that you have changed it twice too. 

Do you have any good ideas?

Reply:
```
* BTN "Cancel" = 取消
* BTN "Close" = 关闭
```
From the above, we can infer that the use of "consent" here is semantically correct.

Reply:
This is more of a "Save"(保存) of some popup options, "OK"(好的/确认), "I know(我知道)", "I agree(我同意)" with buttons like

Reply:
When I used SmartGit 19RC3, I felt that the dialog box at that time was more suitable for "agree"

Reply:
But in the Edit | Preferences popup should be applied("应用") or confirmed(确认), it would be strange to use the "agree(同意)".

Reply:
We have many dialogs using "OK"/"Cancel". Hence, for `*.btn"OK"` I would use a translation which should fit in most cases (but may not be ideal everywhere). For those dialogs, where a different "OK" is more suitable, you can add this explicitly (even if it's not part of master mappings). For example, you can add:

```
*.btn"OK"=好
dlgSgPreferences.btn"OK"=应用
```

Reply:
Because it's used in different environments, I think "OK（确定）" is better.

Reply:
I think it can be, if the separate pop-up window is more suitable, it can be added separately. 
As @mstrap  said like below:

```suggestion
dlgSgPreferences.btn"OK"=应用
```

Reply:
I think the semantics of "确定" is clearer than "应用".

Reply:
What exactly is the state of this PR now? Can it be merged? We are very close to 19.1 GA and I would love to have these new translations included.

Reply:
Please test after merger